cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -190,6 +190,11 @@ CUDF_TYPE_MAPPING(numeric::decimal32, type_id::DECIMAL32);
 CUDF_TYPE_MAPPING(numeric::decimal64, type_id::DECIMAL64);
 CUDF_TYPE_MAPPING(cudf::struct_view, type_id::STRUCT);
 
+template <cudf::type_id Id>
```

Comment:
Could you add a comment explaining how/when to use this?

Reply:
Docs and copyright updated.